src/ts/src/common/components/library/data_subview/index.tsx

```diff
@@ -7,8 +7,9 @@ import { DataTable } from "common/components";
 import style from "./index.module.scss";
 
 interface Props {
-    data?: BioinformaticsData[];
+    data?: Record<string | number, JSONPrimitive>[];
```

Comment:
Why is this moving from BioinformaticsData to this new construct?

Reply:
To make it more generic, if we want to reuse it.

Reply:
`Record` is a built in Typescript type.

Reply:
SGTM!

To keep it DRY, should we declare a `Data` type and use it throughout the file?

E.g.,

```ts
// <DataTable /> src/ts/src/common/components/library/data_table/index.tsx
export type Data = Record<string | number, JSONPrimitive>;

// <DataSubview /> src/ts/src/common/components/library/data_subview/index.tsx
import { DataTable, Data } from 'src/common/components/library/data_table'

interface SearchState {
    searching?: boolean;
    results?: Data[]; // <-- this!
}
...

```